config/initializers/webauthn.rb

```diff
@@ -1,3 +1,3 @@
 WebAuthn.configure do |config|
-  config.algorithms.concat(%w[ES384 ES512 PS256 PS384 PS512])
+  config.algorithms << ["ES384", "ES512", "PS256", "PS384", "PS512"]
```

Comment:
I'm not sure this is right? For a "normal" array, `<<` with an array will add that second array, not extend it. Do we need to use `+=` maybe? The example in the [gem docs](https://github.com/cedarcode/webauthn-ruby#configuration) only `<<`'s one item so it doesn't have this issue

```ruby
x = [1, 2, 3]
# => [1, 2, 3]
x.concat([4, 5, 6])
=> [1, 2, 3, 4, 5, 6] # right

y = [1, 2, 3]
# => [1, 2, 3]
y << [4, 5, 6]
# => [1, 2, 3, [4, 5, 6]] # wrong?
```

Reply:
Yep.  Not right.  Revert...

Reply:
Right!  I knew this at one point :)
Thanks for the 👀